Assets/_BForBoss/_Utility/Scripts/PlayerTriggerHelper.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Perigon.Utility;
+using UnityEngine;
+
+namespace BForBoss.Utility
+{
+    public class PlayerTriggerHelper : MonoBehaviour
```

Comment:
Can call it `PlayerTriggerBehaviour` 

Cuz you can use this with a lot of other things. 

And instead of using an event, can you use a lambda instead? So you don't need to unsub

Reply:
I'm not sure I get what you mean, why wouldn't I need to unsub?

Reply:
Wait so you just mean keep it as an action but remove the event modifier?

Reply:
Yeah